diff --git a/rules/base.xml b/rules/base.xml
index eff2f941..fcddb307 100644
--- a/rules/base.xml
+++ b/rules/base.xml
@@ -6073,6 +6073,12 @@
             <description>English (UK, Colemak)</description>
           </configItem>
         </variant>
+        <variant>
+          <configItem>
+            <name>gb_umlauts</name>
+            <description>English (UK, with Umlauts)</description>
+          </configItem>
+        </variant>
         <variant>
           <configItem>
             <name>colemak_dh</name>
diff --git a/symbols/gb b/symbols/gb
index 3aab5c86..4683b77c 100644
--- a/symbols/gb
+++ b/symbols/gb
@@ -251,3 +251,18 @@ partial alphanumeric_keys
 	xkb_symbols "sun_type6" {
 	include "sun_vndr/gb(sun_type6)"
 };
+
+partial alphanumeric_keys
+   xkb_symbols "gb_umlauts" {
+
+   // Direct access to German umlauts with Multi_key (Alt Gr)
+
+   include "gb(basic)"
+
+   name[Group1]="English (UK, with Umlauts)";
+   key <AC01> { [             a,             A,          adiaeresis,          Adiaeresis ] };
+   key <AD07> { [             u,             U,          udiaeresis,          Udiaeresis ] };
+   key <AD09> { [             o,             O,          odiaeresis,          Odiaeresis ] };
+
+   include "level3(ralt_switch)"
+};
